starter-code/src/components/TotalsCheckout.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default function TotalsCheckout(props) {
+
+    console.log(props)
```

Comment:
Just wanted to mention that you can always check the values of your state and props with your React Dev Tools "Components". Nothing wrong with using console.log, just a friendly reminder ;)